docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/entity-name-is-not-unique.md

```diff
@@ -14,9 +14,9 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-method-is-overridden-on-the-project-level.html
 ---
 
-Modules have public and private APIs. While public API updates always support backward compatibility, private API updates can break backward compatibility. So, backward compatibility is not guaranteed in the private API. For example, if you use a core method on the project level, and it is updated or removed with an update, it can cause unexpected issues.
+The entity name is not unique. The page describes why entities on the project side have to have a unique name (containing some prefix). The project has to follow the rule to be able to correctly receive Spryker autoupdates.
 
-For more information about module APIs, see [Definition of Module API](/docs/scos/dev/architecture/module-api/definition-of-module-api.html).
+During future updates, to Sprykers system could be added entities with any prefixless name (such as new constant, transfer, module). Then auto update put the changes to the vendor of a project. It could provide naming conflict if the same name has existed on the project level.
```

Comment:
```
to Sprykers system could be added entities with any prefixless name (such as new constant, transfer, module)
```
This part is not enough clear specified I think. We spoke about entities names before and now switched to ```any prefixless name```. It can be unclear how does it relate to entities.

Reply:
Thanks 
Replaced to `Not unique names on the project level could provide naming conflict during the auto-update process`